.github/workflows/push_pr.yml

```
@@ -11,21 +11,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s-version: [ "v1.15.12", "v1.16.15", "v1.17.17", "v1.18.16", "v1.19.8", "v1.20.4" ]
+        k8s-version: [ "1.23.15", "1.24.12", "1.25.8", "1.26.1" ]
     env:
       DOCKER_BUILDKIT: '1' # Setting DOCKER_BUILDKIT=1 ensures TARGETOS and TARGETARCH are populated
     steps:
-      - uses: actions/checkout@v2
-      - uses: manusa/actions-setup-minikube@v2.3.0
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.3.0
```

```suggestion
        uses: manusa/actions-setup-minikube@v2.7.2
```
I see that latest release of the [actions-setup-minikube](https://github.com/manusa/actions-setup-minikube) is v2.7.2 can we bump it or is there any reason to stay with version 2.3.0 ?

---

Thanks for catching this. I'll update and do some testing today.